ui/app/adapters/generated-item-list.js

```diff
@@ -4,21 +4,21 @@ import ApplicationAdapter from './application';
 export default ApplicationAdapter.extend({
   namespace: 'v1',
   urlForItem() {},
-  optionsForQuery(id) {
-    let data = {};
-    if (!id) {
-      data['list'] = true;
-    }
-    return { data };
+  optionsForQuery(/* id */) {
```

Comment:
previously, this was only appending the `?list=true` for the url if there was no `id` (via the `queryRecord` method) - if this isn't doing that and we're not using `queryRecord` anywhere in the generated item routes, I'd just remove this method entirely and do `{data: {list: true}}` inline on line 17 below. And if we're not using `queryRecord`, it could be removed from this generated-item-list adapter too.